src/content/changelog/4-60-0-2020-03-23.md

```diff
@@ -0,0 +1,24 @@
+---
+title: '4.60.0: 2020-03-23'
+date: 2020-03-23T04:00:00.000Z
+version: 4.60.0
+changelog:
+  - API
+---
+### Added
+
+- The field `authentication_type` has been added to View Profile ([GET /profile](/api/v4/profile)) and Update Profile ([PUT /profile](/api/v4/profile/#put)). `authentication_type` returns your account's Cloud Manager authentication type. Currently, a user's password (in conjunction with their username) is the only available authentication type.
```

Comment:
My approval only covers this line. Although I reviewed the rest of this Changelog entry, I cannot confirm the accuracy or validity of it.

Reply:
Should we note in here that this field is read only, or is that unnecessary in the changelog because it's sufficiently covered in the doc?